test/jdk/jdk/internal/vm/Continuation/Fuzz.java

```diff
@@ -84,6 +87,10 @@ public class Fuzz implements Runnable {
     static final Path TEST_DIR = Path.of(System.getProperty("test.src", "."));
 
     public static void main(String[] args) {
+        if (Platform.isSlowDebugBuild() && Platform.isOSX() && Platform.isAArch64()) {
```

Comment:
I don't like the idea of skipping the unstable test using SkippedException. Wouldn't be better to add problemlist for slowdebug? So anyone could easy identify test bugs in slowdebug mode. Really it would be better to support bits configurations in standard problem lists like os/arch but it is a separate issue.

Reply:
As far as I know, the ProblemList does not support bits config so there's no way
to specify an entry for 'release' or 'fastdebug' or 'slowdebug' or...

Reply:
Yes, it is needed to make a separate problem list for this and use it in your testing.

The SkippedException and '@requires' are used to filter out the test when it is not applicable for this configuration, not when there is a bug that reproduced only with this configuration. Adding '@requires' usually means that we are not planning to run. 
If you want to add them as exception might be it makes sense to add a corresponding comment.